frontend/packages/kubevirt-plugin/src/components/vm-disks/_create-disk-row.scss

```diff
@@ -0,0 +1,25 @@
+.kubevirt-vm-disks__cell_field--with-addendum {
```

Comment:
classes/rules should be in alphabetical order

Reply:
fixed